src/components/Editor/SearchBar.js

```diff
@@ -223,33 +252,74 @@ const SearchBar = React.createClass({
   },
 
   renderSvg() {
-    const { count, query } = this.state;
+    const { count } = this.state;
 
-    if (count == 0 && query.trim() != "") {
+    if (count == 0 && this.props.query.trim() != "") {
       return Svg("sad-face");
     }
 
     return Svg("magnifying-glass");
   },
 
+  renderSearchModifiers() {
+    const {
+      modifiers: { caseSensitive, wholeWord, regexMatch },
+      toggleModifier } = this.props;
+
+    if (isEnabled("searchModifiers")) {
```

Comment:
@jasonLaster should I also add the feature to some of the configs? I hadn't used this before so I'm not sure if that's also needed (I had to add it to my local.json to test the feature.)